src/Select.js

```diff
@@ -224,7 +232,7 @@ export const defaultProps = {
   pageSize: 5,
   placeholder: 'Select...',
   screenReaderStatus: ({ count }: { count: number }) =>
```

Comment:
Keeping screenReaderStatus for now to not break the existing accessibility API, but i think we should do away with this in favor of the more verbose pattern specified above.